src/main/java/hudson/tasks/junit/TestResult.java

```diff
@@ -178,51 +198,60 @@ public TestResult getTestResult() {
     }
 
     @Deprecated
-    public void parse(long buildTime, DirectoryScanner results) throws IOException {
-        parse(buildTime, results, null);
+    public void parse(long filesTimestamp, DirectoryScanner results) throws IOException {
+        parse(filesTimestamp, results, null);
     }
 
     /**
      * Collect reports from the given {@link DirectoryScanner}, while
      * filtering out all files that were created before the given time.
-     * @param buildTime Build time.
+     * @param filesTimestamp file timestamp to filter files older.
```

Comment:
Renaming variables like this makes it harder to review the actually changed behavior.

Reply:
frankly the previous name was just not really accurate

Reply:
Probably so, but it would make your PR easier to understand and review if that were kept to a separate follow-up `chore` PR that claims to not change behavior.